lyndapham/step/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,111 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+  /** This class finds available meeting times given a MeetingRequest */
```

Comment:
The class comment should be in the line before the class definition.

Reply:
I've moved it!